gripql/python/gripql/query.py

```diff
@@ -297,9 +298,8 @@ def __stream(self, debug=False):
         logger.debug('POST %s', self.url)
         logger.debug('BODY %s', self.to_json())
         logger.debug('STATUS CODE %s', response.status_code)
-        raise_for_status(response)
 
-        for result in response.iter_lines(chunk_size=None):
+        for result in response.iter_lines():
```

Comment:
What is the effect of removing `chunk_size=None` ?

Reply:
From requests ( https://2.python-requests.org/en/master/api/#requests.Response.iter_content ): `chunk_size must be of type int or None. A value of None will function differently depending on the value of stream. stream=True will read data as it arrives in whatever size the chunks are received. If stream=False, data is returned as a single chunk.`

Reply:
It defaults to 512 with `iter_lines`. I'll test a few different values to see how they effect speed. It would probably be most noticeable on large lines, line GeneExpression vectors.

Reply:
I tested it, and chunk_size=None seems to decrease CPU usage and wall time, so I've reverted to original